spec/sidekiq/job_logger_spec.rb

```diff
@@ -0,0 +1,119 @@
+require 'spec_helper'
+
+RSpec.describe Loga::Sidekiq::JobLogger do
+  subject(:job_logger) { described_class.new }
+
+  let(:target) { StringIO.new }
+
+  let(:json_line) do
+    target.rewind
+    JSON.parse(target.read.split("\n").last)
+  end
+
+  before do
+    Loga.reset
+
+    Loga.configure(
+      service_name: 'hello_world_app',
+      service_version: '1.0',
+      device: target,
+      format: :gelf,
+    )
+  end
+
+  describe '#call' do
+    context 'when the job passess successfully' do
+      let(:item_data) do
+        {
+          'class' => 'HardWorker',
+          'args' => ['asd'],
+          'retry' => true,
+          'queue' => 'default',
+          'jid' => '591f6f66ee0d218fb451dfb6',
+          'created_at' => 1_528_799_582.904939,
+          'enqueued_at' => 1_528_799_582.9049861,
+        }
+      end
+
+      it 'has the the required attributes on call' do
+        job_logger.call(item_data, 'queue') do
+          puts 'Lorem Ipsum dollor'
```

Comment:
This goes to `stdout`. it will be better if you just leave a comment `# doing something`